### PR TITLE
executive summary budget

### PR DESCRIPTION
**Pull request by brendansudol**
### This pull request changes...
- adds executive summary budget tables

resolves: https://github.com/18F/cms-hitech-apd/issues/626

preview:
https://cl.ly/sOqV

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author

### This feature is done when...
- [x] Product has approved the experience

### CODE REVIEW

**Comment by NAretakis** [comment]
Looking good. For the HIT/HIE table, we need change some of column entries to state share, so that we have both Federal and State share displayed.

Here is the wireframe for HIT/HIE:
image

For MMIS we want ouse the formatting from the wireframe above, but use the columns/data from this table: 
image

For all of these, their data should be driven by the federal and state share amounts from the cost allocation table. Actually... this is a little complex, let's have a quick conversation @brendansudol

File: web/src/containers/ExecutiveSummaryBudget.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { t } from '../i18n';
+import { formatMoney } from '../util/formats';
+
+const DollarCell = ({ value }) => (
+  <td className="mono right-align">{formatMoney(value)}</td>
+);
```
Comment (mgwalker) [inline web/src/containers/ExecutiveSummaryBudget.js]: `<TDAmeriDollar />` :trollface:

File: web/src/containers/ExecutiveSummaryBudget.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              <th className="right-align">Federal share</th>
+              <th className="right-align">Federal share</th>
+              <th className="right-align">Grand total</th>
+            </tr>
+          </thead>
+          <tbody>
+            {rowKeys.map(({ year, display }) => (
+              <tr key={year}>
+                <td>{display}</td>
+                <DollarCell value={hitAndHie.combined[year].federal} />
```
Comment (mgwalker) [inline web/src/containers/ExecutiveSummaryBudget.js]: These should be MMIS, right?

**Comment by NAretakis** [comment]
This PR is on hold until #685 is resolved.

**Comment by mgwalker** [comment]
Should this still be held pending #685 (addressed in PR #692), or should we merge it now and fix it afterwards?  Don't want to leave it sitting too long as that increases the risk that changes here will conflict with other changes that are made afterwards.

**Comment by brendansudol** [comment]
now that https://github.com/18F/cms-hitech-apd/pull/692 is in 🎉, i will adjust this bad boy

**Comment by brendansudol** [comment]
alright @mgwalker and @NAretakis -- this is ready for a re-review :)

**Pull request merged**